man/org.freedesktop.systemd1.xml

```diff
@@ -1073,27 +1075,27 @@ $ gdbus introspect --system --dest org.freedesktop.systemd1 \
       last watchdog ping received from the service, or 0 if none was ever received.</para>
 
       <para><varname>ExecStartPre</varname>, <varname>ExecStart</varname>, <varname>ExecStartPost</varname>,
-      <varname>ExecReload</varname>, <varname>ExecStop</varname>, <varname>ExecStop</varname> each are arrays
-      of structures each containing: the binary path to execute; an array with all arguments to pass to the
-      executed command, starting with argument 0; a boolean whether it should be considered a failure if the
-      process exits uncleanly; two pairs of
+      <varname>ExecReload</varname>, <varname>ExecStop</varname>, and <varname>ExecStop</varname> are arrays
+      of structures where each struct contains: the binary path to execute; an array with all arguments to
+      pass to the executed command, starting with argument 0; a boolean whether it should be considered a
+      failure if the process exits uncleanly; two pairs of
       <constant>CLOCK_REALTIME</constant>/<constant>CLOCK_MONOTONIC</constant> microsecond timestamps when
       the process began and finished running the last time, or 0 if it never ran or never finished running;
       the PID of the process, or 0 if it has not run yet; the exit code and status of the last run. This
       field hence maps more or less to the corresponding setting in the service unit file but is augmented
       with runtime data.</para>
 
       <para><varname>LimitCPU</varname> (and related properties) map more or less directly to the
-      corresponding settings in the service unit files, however are set to 18446744073709551615 (i.e. -1) if
-      they aren't set.</para>
+      corresponding settings in the service unit files However, these are set to 18446744073709551615
+      (i.e. -1) if they aren't set.</para>
```

Comment:
Hmm, maybe reword this, while at it. "these are set to ... if they aren't set" is a bit self-contradictory ;)

Reply:
I went with "except that if they aren't set, their value is 18446744073709551615 (i.e. -1).". Maybe there's a better word than "is" though. Although using "is set to" is weird because of the self-contradiction.